lib/remote/apilistener.hpp

```diff
@@ -71,6 +71,22 @@ enum class ApiCapabilities : uint_fast64_t
 	IfwApiCheckCommand = 1u << 1u,
 };
 
+class ConfigSyncRule
```

Comment:
I don't really get the name here, why "rule"? Maybe something like `ObjectNameMutex`, as it provides/emulates a mutex per object name.